src/app/components/pages/AdminUserManager.tsx

```diff
@@ -293,6 +293,7 @@ export const AdminUserManager = () => {
                                             <th>User role</th>
                                             <th>School</th>
                                             <th>Verification status</th>
+                                            <th>Teacher pending?</th>
```

Comment:
is the meaning of this clear to the end user?

Reply:
I could add one of the tool tips with an explanation on hover?